### ISSUE TITLE
Control file for 3fd not found

### ISSUE THREAD

**Issue by mrexodia** [cmpute/vcpkg-gui#7]
![image](https://user-images.githubusercontent.com/2458265/116818319-12314000-ab6b-11eb-81cb-85b20f6a948e.png)

![image](https://user-images.githubusercontent.com/2458265/116818333-2c6b1e00-ab6b-11eb-8732-30aa51b64606.png)

Likely due to changes in the way vcpkg works.

**Comment by cmpute** [comment]
Yeah.. I haven't worked on this project for quite a long time, there are some break changes for vcpkg
